desmume/src/frontend/windows/main.cpp

```diff
@@ -2961,6 +2961,46 @@ void WavEnd()
 	NDS_UnPause();
 }
 
+void UpdateTitle(const char* curr_title)
+{
+	if (gameInfo.hasRomBanner())
+	{
+		if (curr_title == nullptr) {
+			curr_title = EMU_DESMUME_NAME_AND_VERSION();
+		}
+
+		char new_title[512];
+		char game_title[128];
+
+		strcpy(new_title, curr_title);
+
+		int new_lenght = strlen(new_title);
```

Comment:
Hate to be that guy but you spelt "length" wrong. Oh and while this file is a mess, variables are mostly camelCase. I'm not a maintainer so you can ignore what I said but it kind of triggered me.

Reply:
Sorry, fixed